src/tensor_ops.jl

```diff
@@ -241,6 +241,17 @@ Computes the deviatoric part of a second order tensor.
     end
 end
 
+###################
+# Permute indices #
+###################
+
+function permute_index{dim}(S::FourthOrderTensor{dim},idx::NTuple{4,Int})
```

Comment:
Yea I know,
Initially wrote this to use it with minor/major transpose. But realized the specialized edition which exist now are much better, decided to keep this anyway.